templates/tfengine/components/project/terraform.tfvars

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{hclField . "exists"}}
+project_id = "{{.project_id}}"
+parent_type = "{{.parent_type}}"
+parent_id = "{{.parent_id}}"
+billing_account = "{{.billing_account}}" 
```

Comment:
Because they're always present.

It is `required: false` on the schema because they're provided at a higher level `data {}` block.

Should you consider it valuable to put them as `hclField`? I tried and doesn't change anything

Reply:
Yes, I think using the function here makes the code more consistent.